src/util/environment.cpp

```diff
@@ -35,7 +36,14 @@ void unsetEnvVar(const std::string& varName)
 
 unsigned int getUsableCores()
 {
-    unsigned int nCores = std::thread::hardware_concurrency();
+    auto conf = faabric::util::getSystemConfig();
+    unsigned int nCores;
+
+    if (conf.overrideCpuCount == 0) {
+        nCores = std::thread::hardware_concurrency();
+    } else {
+        nCores = conf.overrideCpuCount;
+    }
```

Comment:
Can you add a test for this? Mainly to check that overriding returns the overridden value rather than the default.

Reply:
I completely forgot about `/util` tests. Sorry about that.